src/cactus.js

```diff
@@ -134,28 +74,26 @@ async function cutReleaseBranch(args) {
 
   // Shell out to run npm version inside tempdir
   cp.execSync(`npm version ${args.level} -m "Release v%s"`, { cwd: tmpdir.name });
-  repo.refreshIndex();
 
   if (!approval.lgtm) {
     return 'Aborted branch cut! Phew, that was a close one...';
   }
 
   // Push master, the release branch, and tag
   logger.info(`Pushing branch ${versionInfo.releaseBranchName} & tag v${versionInfo.version}`);
-  await clonedRemote.push([
-    `refs/heads/master:refs/heads/master`,
-    `refs/heads/master:refs/heads/${versionInfo.releaseBranchName}`,
-    `refs/tags/v${versionInfo.version}:refs/tags/v${versionInfo.version}`,
-  ], makeGitOpts(credentials));
+
+  await clonedRepo.push('origin', 'master');
+  await clonedRepo.push('origin', `master:${versionInfo.releaseBranchName}`);
+  await clonedRepo.pushTags('origin');
+  await repo.pull();
```

Comment:
comment that this is original repo.

This should be the equivalent of `git pull --rebase`